code/go/main-workshop/cdk-workshop_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/aws/aws-cdk-go/awscdk/v2"
+	assertions "github.com/aws/aws-cdk-go/awscdk/v2/assertions"
```

Comment:
I suspect the package is already named "assertions", so there is no need to re-alias it.

```suggestion
	"github.com/aws/aws-cdk-go/awscdk/v2/assertions"
```